cms/static/sass/xmodule/_image-zoom-tool.scss

```diff
@@ -0,0 +1,64 @@
+// The LMS version is in:
+// /lms/static/sass/course/_image-zoom-tool.scss
+
+// edX image zoom tool
+.zooming-image-place {
```

Comment:
Instead of creating separate CMS and LMS sass files, can you find a location in common to put a single file?